Source/objects.cpp

```diff
@@ -4073,8 +4073,8 @@ bool OperateShrineSecluded(int pnum)
 		return true;
 
 	for (int yy = 0; yy < DMAXY; yy++) {

```

Comment:
This can be an `std::fill_n(automapview, DMAXX * DMAXY)`

Reply:
How about the ones in loadsave.cpp? They need to happen in the same order as the data is packed in to the save game.

Reply:
Went with `std::fill(&automapview[0][0], &automapview[DMAXX][DMAXY], true);` just to feel like I'm a free spirite.